internal/npm_install/generate_pnp_build_files.ts

```diff
@@ -0,0 +1,44 @@
+import * as fs from "fs";
+import * as path from "path";
+
+const workspacePath = process.argv.slice(2)[0];
+
+function mkdirp(p: string) {
+  if (!fs.existsSync(p)) {
+    mkdirp(path.dirname(p));
+    fs.mkdirSync(p);
+  }
+}
+
+function writeFileSync(p: string, content: string) {
```

Comment:
Not necessarily, but I didn't particularly want to deal with callbacks, but I see now fs has a promises property in node@^10, so it might be better to just use that